ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -54,6 +54,66 @@ ethernet_interfaces:
     <...>
 ```
 
+## Removal Ansible tags from AVD roles
+
+Support for Ansible tags across AVD roles has been replaced with variables to control this execution of various parts of AVD.
+
+The previous implementation of Ansible tags was not consistent and mostly undocumented. It often lead to inconsistent behaviors depending
+when the tags were used with `--tags` vs. `--skip-tags`.
+
+AVD now supports a number of variables which allow detailed control of which areas of AVD should be executed.
+Variables are only added for the roles providing multiple optional functionalities.
+
+| Ansible Role | Variable | Default value | Description |
+| ------------ | -------- | ------------- | ----------- |
+| All | `avd_verify_requirements` | `true` | Verify Ansible and Python requirements. |
+| All | avd_create_directories | `true` | Create required output directories if not present. |
+| `arista.avd.eos_designs` | `avd_eos_designs_debug` | `false` | Dump all vars and facts per device after generating `avd_switch_facts`. |
+| `arista.avd.eos_designs` | `avd_eos_designs_structured_config` | `true` | Generate structured configuration per device. |
```

Comment:
I suggest cutting this option. The documentation won't work if you don't generate the structured config.

Reply:
The documentation now reads the structured configs from the files instead of hostvars. So it should be possible to generate docs for the fabric.

Reply:
agreed, and fixed bug!